MoreCyclopsUpgrades/Modules/Recharging/Nuclear/NuclearChargingManager.cs

```diff
@@ -31,6 +31,15 @@ private static InventoryItem SpawnDepletedRod()
             Pickupable pickupable = gameObject.GetComponent<Pickupable>().Pickup(false);
             return new InventoryItem(pickupable);
         }
+
+        private static InventoryItem SpawnDepletedModule()
+        {
+            GameObject prefab = Resources.Load<GameObject>("WorldEntities/Natural/DepletedReactorRod");
```

Comment:
This should be calling to the new prefab you created for the depleted module item

Reply:
Fixed